app/src/main/java/ch/hikemate/app/ui/guide/GuideScreen.kt

```diff
@@ -0,0 +1,286 @@
+  Text(
+      text = stringResource(R.string.guide_hiking_section_title),
+      style = MaterialTheme.typography.headlineMedium)
+  Spacer(modifier = Modifier.height(GuideScreen.HEADER_SPACING_DP.dp))
+}
+
+/**
+ * The composable function for a single topic card
+ *
+ * @param topic the topic which will be described
+ * @param navigationActions
+ */
+@Composable
+private fun ExpandableTopicCard(
+    topic: GuideTopic,
+    navigationActions: NavigationActions,
+) {
+  var isExpanded by remember { mutableStateOf(false) }
+
+  Card(
+      modifier =
+          Modifier.fillMaxWidth()
+              .padding(vertical = GuideScreen.CARD_SPACING_DP.dp)
+              .animateContentSize()
+              .testTag("${GuideScreen.TOPIC_CARD}_${topic.titleResId}"),
+      shape = RoundedCornerShape(GuideScreen.CORNER_RADIUS_DP.dp)) {
+        TopicCardContent(
+            topic = topic,
+            isExpanded = isExpanded,
+            onExpandToggle = { isExpanded = !isExpanded },
+            navigationActions = navigationActions)
+      }
+}
+
+@Composable
+private fun TopicCardContent(
+    topic: GuideTopic,
+    isExpanded: Boolean,
+    onExpandToggle: () -> Unit,
+    navigationActions: NavigationActions
+) {
+  Column {
+    // Move clickable to just the header
+    Row(
+        modifier =
+            Modifier.fillMaxWidth()
+                .clickable(onClick = onExpandToggle)
+                .testTag("${GuideScreen.TOPIC_HEADER}_${topic.titleResId}")
+                .padding(GuideScreen.CONTENT_PADDING_DP.dp),
+        horizontalArrangement = Arrangement.SpaceBetween,
+        verticalAlignment = Alignment.CenterVertically) {
+          Text(
+              text = stringResource(topic.titleResId), style = MaterialTheme.typography.titleMedium)
+          Icon(
+              imageVector =
+                  if (isExpanded) Icons.Default.KeyboardArrowUp
+                  else Icons.Default.KeyboardArrowDown,
+              contentDescription =
+                  stringResource(
+                      if (isExpanded) R.string.collapse_topic else R.string.expand_topic))
+        }
+    ExpandableContent(topic = topic, isExpanded = isExpanded, navigationActions = navigationActions)
+  }
+}
+
+@Composable
+private fun ExpandableContent(
+    topic: GuideTopic,
+    isExpanded: Boolean,
+    navigationActions: NavigationActions
+) {
+  AnimatedVisibility(
+      modifier = Modifier.testTag("${GuideScreen.TOPIC_CONTENT}_${topic.titleResId}"),
+      visible = isExpanded,
+      enter = AnimationConfig.enterTransition,
+      exit = AnimationConfig.exitTransition) {
+        Column(modifier = Modifier.padding(GuideScreen.CONTENT_PADDING_DP.dp)) {
+          Text(
+              text = stringResource(topic.contentResId), style = MaterialTheme.typography.bodyLarge)
+
+          Spacer(modifier = Modifier.height(GuideScreen.HEADER_SPACING_DP.dp))
+
+          topic.actionRoute?.let { route ->
```

Comment:
Praise: I like a lot this feature of adding a button, so the user can immediately try out what they just learned!